docs/program/sdks/_index.md

```diff
@@ -215,19 +253,27 @@ Find documentation on how to use these methods here:
 
 - [Arm](/components/arm/#api)
 - [Base](/components/base/#api)
+- [Board](/components/board/#api)
 - [Camera](/components/camera/#api)
+- [Encoder](/components/encoder/#api)
 - [Gantry](/components/gantry/#api)
 - [Gripper](/components/gripper/#api)
 - [Input Controller](/components/input-controller/#api)
 - [Motor](/components/motor/#api)
 - [Movement Sensor](/components/movement-sensor/#api)
 - [Sensor](/components/sensor/#api)
 - [Servo](/components/servo/#api)
-<!--
-Board API Docs should be added soon on Viam Documentation, + Encoder
--->
 
-You can find example code in the [Python SDK example GitHub repository](https://github.com/viamrobotics/viam-python-sdk/tree/main/examples/server/v1), the [Golang SDK example GitHub repository](https://github.com/viamrobotics/rdk/tree/main/examples), or the [TypeScript SDK example GitHub repository](https://github.com/viamrobotics/viam-typescript-sdk/tree/main/examples).
+{{% alert title="Tip" color="tip" %}}
+
+You can find example code using the SDKs in the <file>examples</file> folder of the respective repository:
```

Comment:
nit: the `python` example takes us to a few subdirectories within the `examples` directory, while all other links take us directly to the `examples` directory. None of the four really require the additional step "then navigate to the `examples` folder" that this list intro seems to suggest. Perhaps just plainly stating without monospace helps to be precise for all four, something like:

> Each SDK provides example code in the following locations:

Or similar.

Reply:
```suggestion
Each SDK provides example code in its repository:
```